Microsoft.Toolkit.Uwp.SampleApp/SamplePages/BladeControl/BladePage.xaml

```diff
@@ -33,15 +32,19 @@
                 </VisualState>
             </VisualStateGroup>
         </VisualStateManager.VisualStateGroups>
+        
         <controls:BladeControl x:Name="BladeControl"
-                               Padding="24"
+                               Padding="0"
                                HorizontalAlignment="Stretch"
-                               VerticalAlignment="Stretch">
+                               VerticalAlignment="Stretch"
+                               FullScreenBlades="True">
```

Comment:
This should be changed either by a new binding button (part of the .bind file) or through an AdaptiveTrigger based on width of the screen.